modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/bnd.bnd

```diff
@@ -36,6 +36,6 @@ Import-Package:\
 	org.tukaani.*;resolution:=optional,\
 	\
 	*
-Liferay-Require-SchemaVersion: 5.3.1
+Liferay-Require-SchemaVersion: 5.3.2
```

Comment:
@alberttgomes, can you please adjust the commit message to be like: `Update schema version`?

Once your commit is only adjusting the schema version of the module `dynamic-data-mapping-service` then there is no need to emphasize it on the commit message, does it make sense to you?